assignments/array-methods.js

```diff
@@ -56,22 +56,34 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(runnerID){
+    fullName.push(runnerID.first_name + " " + runnerID.last_name);
+});
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map(runnerID => {
+   allCaps.push(runnerID.first_name.toUpperCase());
```

Comment:
In this scenario, you could assign let allCaps to  the runners.map and get rid of the .push() being as map already creates a new array.